File: meetupguider-api/src/MeetupGuilder.Entities/Models/CountryTopic.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace MeetupGuilder.Entities.Models
{
    [Table("CountryTopics")]
    public class CountryTopic
    {
        public int Id { get; set; }
        public string Country { get; set; }
        public string City { get; set; }
        public float Lon { get; set; }
        public float Lat { get; set; }
        public string TopicName { get; set; }
        public int TopicCount { get; set; }
    }
}

File: meetupguider-ui/src/features/heatmap-overlay/types/country-topic.ts
class CountryTopic {
    country: string;

    city: string;

    lat: number;

    lon: number;

    topicName: string;

    topicCount: number;

    constructor(country: string, city: string, lat: number, lon: number, topicName: string, topicCount: number) {
      this.country = country;
      this.city = city;
      this.lat = lat;
      this.lon = lon;
      this.topicCount = topicCount;
      this.topicName = topicName;
    }
}

export default CountryTopic;
